docs/deployment/engine_on_yarn.md

```diff
@@ -175,6 +175,24 @@ If the `TopSpeedWindowing` passes, configure it in `$KYUUBI_HOME/conf/kyuubi-env
 $ echo "export HADOOP_CONF_DIR=/path/to/hadoop/conf" >> $KYUUBI_HOME/conf/kyuubi-env.sh
 ```
 
+#### Required Environment Variable
+
+The `FLINK_HADOOP_CLASSPATH` is required, too. It should contain `commons-collections-*.jar`, 
+`hadoop-client-runtime-*.jar`, `hadoop-client-api-*.jar` and `htrace-core4-*.jar`.
+All four jars are in the `HADOOP_HOME`. 
```

Comment:
I would say this is not accurate. The `FLINK_HADOOP_CLASSPATH` should ponit to Hadoop client localtion, generally it compose of Hadoop client configuration files, e.g. `core-site.xml`, `yarn-site.xml`, and Hadoop client jars, for Hadoop 3, Hadoop shaded client is recommended instead of vanilla jars.

Reply:
Is it more reasonable to set to `hadoop classpath`?

Reply:
It should be. But `hadoop classpath` always ponits to vanilla jars I think, user may suffer jar hell then.

Reply:
Some background https://issues.apache.org/jira/browse/HADOOP-11656

Reply:
And as a downstream project, Spark always use Hadoop shaded client since Spark 3.2.0.

Reply:
ok i'll take a look and fix it